traffic_portal/test/integration/specs/DeliveryServiceRequest.spec.ts

```diff
@@ -0,0 +1,85 @@
+            })
+            xit('can login', async function(){
+                browser.get(browser.params.baseUrl);
+                await loginPage.Login(login);
+                expect(await loginPage.CheckUserName(login)).toBeTruthy();
+                await deliveryServiceRequestPage.OpenServicesMenu();
+            })
+            deliveryServiceRequestData.create.forEach(create => {
+                xit(create.description, async () => {
+                    await deliveryServiceRequestPage.OpenDeliveryServicePage();
+                    expect(await deliveryServiceRequestPage.CreateDeliveryServiceRequest(create)).toBe(true);
+                    await deliveryServiceRequestPage.OpenDeliveryServiceRequestPage();
+                    await deliveryServiceRequestPage.SearchDeliveryServiceRequest(create);
+                    expect(await deliveryServiceRequestPage.FullFillDeliveryServiceRequest(create)).toBe(true);
+                    await deliveryServiceRequestPage.OpenDeliveryServiceRequestPage();
+                    await deliveryServiceRequestPage.SearchDeliveryServiceRequest(create);
+                    expect(await deliveryServiceRequestPage.CompleteDeliveryServiceRequest(create)).toBe(true);
+
+                });
+            });
+            deliveryServiceRequestData.remove.forEach(remove =>{
+                xit(remove.description, async () =>{
+                    await deliveryServiceRequestPage.OpenDeliveryServicePage();
+                    expect(await deliveryServiceRequestPage.CreateDeliveryServiceRequest(remove)).toBe(true);
+                    await deliveryServiceRequestPage.OpenDeliveryServiceRequestPage();
+                    await deliveryServiceRequestPage.SearchDeliveryServiceRequest(remove);
+                    expect(await deliveryServiceRequestPage.DeleteDeliveryServiceRequest(remove)).toBe(true);
+                })
+            })
+            deliveryServiceRequestData.update.forEach(update=>{
+                xit(update.description, async () =>{
+                    await deliveryServiceRequestPage.OpenDeliveryServicePage();
+                    expect(await deliveryServiceRequestPage.CreateDeliveryServiceRequest(update)).toBe(true);
+                    await deliveryServiceRequestPage.OpenDeliveryServiceRequestPage();
+                    await deliveryServiceRequestPage.SearchDeliveryServiceRequest(update);
+                    expect(await deliveryServiceRequestPage.UpdateDeliveryServiceRequest(update)).toBe(true);
+                    expect(await deliveryServiceRequestPage.FullFillDeliveryServiceRequest(update)).toBe(true);
+                    await deliveryServiceRequestPage.OpenDeliveryServiceRequestPage();
+                    await deliveryServiceRequestPage.SearchDeliveryServiceRequest(update);
+                    expect(await deliveryServiceRequestPage.CompleteDeliveryServiceRequest(update)).toBe(true);
+                })
+            })
+        });
+    });
+});
```

Comment:
Because the DSR is unable to be turn on and off in Github Actions. So right now, test is created just for migrate from E2E to integration and user have options to run it or not like E2E.

Reply:
Hm. That means that the GHA passing on this PR doesn't actually run these tests, huh? I'll try to get 'em running locally, but ultimately we need to figure out how to get that running so we don't break things. That's the only way tests are actually useful (because I guarantee nobody is running these tests prior to opening a PR for Traffic Portal)

Reply:
@mitchell852